target/linux/ath79/image/nand.mk

```diff
@@ -231,6 +231,23 @@ define Device/netgear_ath79_nand
   UBINIZE_OPTS := -E 5
 endef
 
+define Device/netgear_pgzng1
+  SOC := ar9344
+  DEVICE_MODEL := PGZNG1
+  DEVICE_VENDOR := NETGEAR
+  DEVICE_PACKAGES := kmod-usb2 kmod-usb-ledtrig-usbport kmod-i2c-gpio \
+  kmod-leds-pca955x kmod-rtc-isl1208 kmod-spi-dev uboot-envtools
+  KERNEL_SIZE := 5120k
+  IMAGE_SIZE := 83968k
+  PAGESIZE := 2048
+  BLOCKSIZE := 128k
+  KERNEL := kernel-bin | append-dtb | lzma | uImage lzma | \
+    check-size $$(KERNEL_SIZE)
```

Comment:
The check-size is not needed. For KERNEL, the ```Device/Build/kernel``` function will perform a check-size automatically.